src/PythonWrapper.cpp

```diff
@@ -134,6 +123,27 @@ void *PyCreateProducer(const char *groupId) {
     PyEval_InitThreads();  // ensure create GIL, for call Python callback from C.
     return (void *) CreateProducer(groupId);
 }
+
+void *PyCreateTransactionProducer(const char *groupId, PyObject *localTransactionCheckerCallback) {
+    PyEval_InitThreads();
+    CProducer *producer = CreateTransactionProducer(groupId, &PyLocalTransactionCheckerCallback, NULL);
```

Comment:
I can't find `CreateTransactionProducer` implementation in the cpp repository, only the function prototype was found.